src/Currency/CurrencyServiceProvider.php

```diff
@@ -44,7 +44,7 @@ public function registerCurrency()
     {
         $this->app->singleton('currency', function ($app) {
 
-            $this->app['events']->fire('currency.beforeRegister', [$this]);
+            $this->app['events']->dispatch('currency.beforeRegister', [$this]);
```

Comment:
The fire method was reinstated in IgniterFlameEventsDispatcher for backward compatibility, so fire should still work here as long as the instance of the events within the container is the same one from the flame library.

Reply:
Yep, should have said this is coming from the composer package work. So app events wont be a flame app.